### ISSUE TITLE
Undefined symbols for architecture arm64 - ShareLinkActivity

### ISSUE THREAD

**Issue by gald89** [alexruperez/ARFacebookShareKitActivity#1]
I was trying to get ARFacebookShareKitActivity to work in my Objective-C project using the following steps:
1. Installed ARFacebookShareKitActivity through cocoa pods using the following command `pod "ARFacebookShareKitActivity", '~> 1.1.1'`
2. Converted to Swift 3.0 syntax using the Xcode 8 prompt.
3. Added the bridging header `#import "ARFacebookShareKitActivity-Swift.h"`
4. Create a _ShareLinkActivity_ instead of a _UIActivity_ using    

```
NSArray *activityItems = [NSArray arrayWithObjects:someString, someImageData, someUrl, nil];
ShareLinkActivity *shareLinkActivity = [[ShareLinkActivity alloc]init];
UIActivityViewController *activityViewController = [[UIActivityViewController alloc] initWithActivityItems:activityItems applicationActivities:@[shareLinkActivity]]; 
```
- After attempting to build I got a 'Undefined symbols for architecture arm64' and a reference to the ShareLinkActivity class.
- Even after changing all valid architecture to 'arm v7, arm 7 and arm64', setting the build active target only to False and adding various flags, I still could not get this to work.

**Comment by alexruperez** [comment]
Hi @gald89, here some questions:
- Are your using use_frameworks! in your Podfile?
- Are you using the last CocoaPods gem version?
- Are you using Swift 3.0 in your project? Remember 1.1.1 for Swift 3.0 or fix the version to 1.1 for Swift 2.1
- If your Architectures and Valid Architectures are all right, you may check whether you have add $(inherited) to Other Linker Flags.

**Comment by gald89** [comment]
- Yes
- CocoaPods gem version 1.0.1
- No but I am using Xcode 8 so in order to build I need to convert to Swift 2.3 or 3.0
- $(inherited) was also added to Other Linker Flags

**Comment by alexruperez** [comment]
@gald89 fixed for release 1.2
